docs/cli.rst

```diff
@@ -1,11 +1,247 @@
+* The syntax for running the command
+* Any options for the task
+
+Information on specific options includes:
+
+* The syntax for the option (``-o option_name value``).
+* If the options is required or optional.
+* A description of each option.
+
+Example output looks like this:
+
+.. code-block:: console
+
+    $ cci task info util_sleep
+    util_sleep
+
+    Description: Sleeps for N seconds
+
+    Class: cumulusci.tasks.util.Sleep
+
+    Command Syntax
+
+        $ cci task run util_sleep
+
+    Options
+
+        -o seconds SECONDS
+        Required
+        The number of seconds to sleep
+        Default: 5
+
+For additional information on flows use::
+
+    $ cci flow info <flow_name>
+
+Where ``<flow_name>`` is the actual name of the flow.
+Information on specific flows includes:
+
+* A description of the flow.
+* The ordered steps (and substeps) of a flow.
+
+Example output looks like this:
+
+.. code-block:: console
+
+    $ cci flow info dev_org
+    Description: Set up an org as a development environment for unmanaged metadata
+    1) flow: dependencies [from current folder]
+        1) task: update_dependencies
+        2) task: deploy_pre
+    2) flow: deploy_unmanaged
+        0) task: dx_convert_from
+        when: project_config.project__source_format == "sfdx" and not org_config.scratch
+        1) task: unschedule_apex
+        2) task: update_package_xml
+        when: project_config.project__source_format != "sfdx" or not org_config.scratch
+        3) task: deploy
+        when: project_config.project__source_format != "sfdx" or not org_config.scratch
+        3.1) task: dx_push
+            when: project_config.project__source_format == "sfdx" and org_config.scratch
+        4) task: uninstall_packaged_incremental
+        when: project_config.project__source_format != "sfdx" or not org_config.scratch
+    3) flow: config_dev
+        1) task: deploy_post
+        2) task: update_admin_profile
+    4) task: snapshot_changes
+
+
+
+Troubleshooting Errors
+----------------------
+
+Errors happen! That’s why our team strives to provide our users with options for efficiently working with them when they occur.
+
+The ``--debug`` Option
+^^^^^^^^^^^^^^^^^^^^^^
+All CumulusCI commands can be passed the ``--debug`` option. When this is used, the following occurs:
+    * Any calls to CumulusCI's logger at the debug level are shown.
+    * Any errors captured by ``requests.packages.urllib3.add_stderr_logger()`` are shown.
+    * If an error is present, the corresponding stacktrace is shown. 
+    * The user is dropped into a `post-mortem debugging <https://docs.python.org/3/library/pdb.html#pdb.post_mortem>`_ session.
+
+Log Files
+^^^^^^^^^
+CumulusCI creates a log file every time a cci command besides ``gist`` is run. There are six rotating logfiles (``cci.log, cci.log1...5``) with ``cci.log`` being the most recent. Logfiles are stored under ``~/.cumulusci/logs``. By default, logfiles capture the the following:
```

Comment:
Be consistent with "log file" vs "logfile". I'd go with "log file"

I'd also leave out "besides gist". It's true but more of a distracting detail.